src/components/homepage/HomePage.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import Login from "./login/Login";
+import SignUp from "./signup/SignUp";
+
+class HomePage extends Component {
+
+    state = {
+        loadLoginPage: false,
+        loadSignUpPage: false
+    }
+
+    handleLogin = () => {
+        this.setState({loadLoginPage: true})
+
+    }
+
+    handleSignUp = () => {
+        this.setState({loadSignUpPage: true})
+    }
+
+    render() {
+        const {loadLoginPage, loadSignUpPage} = this.state;
+        let pageToRender = <div>Testing Something</div>;
```

Comment:
Please remove the "Testing Something" and put only an empty div there or null.

Reply:
done